src/Compilers/CSharp/Portable/Parser/Lexer.Interpolation.cs

```diff
@@ -2,28 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using Microsoft.CodeAnalysis.Text;
+
 namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
 {
     internal partial class Lexer
     {
         internal readonly struct Interpolation
         {
-            public readonly int OpenBracePosition;
-            public readonly int ColonPosition;
-            public readonly int CloseBracePosition;
-            public readonly bool CloseBraceMissing;
+            public readonly Range OpenBraceRange;
```

Comment:
ranges make everything so much nicer and easier to work with.

Reply:
they're also important for raw-string literals where the open brace may be somethin glike `{{{` not `{`.